lib/sync-srv.js

```diff
@@ -409,6 +406,7 @@ function returnUpdates(dataset_id, params, resIn, cb) {
     }
 
     if (!resIn) {
+      console.log('Creating resIn');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)